distros/ros2-overlay.nix

```
@@ -77,6 +77,12 @@ rosSelf: rosSuper: with rosSelf.lib; {
     ];
   });
 
+  ompl = rosSuper.ompl.overrideAttrs ({
+    ...
+  }: {
+    patches = [ ./ompl_ros2.patch ];
```

One more thing (sorry for not noticing this earlier): You should append to existing patches (if they are added in the future).
```suggestion
    patches ? [], ...
  }: {
    patches = patches ++ [ ./ompl_ros2.patch ];
```

---

fixed.